playbook/app/pb_kits/playbook/pb_date/docs/_description.md

```diff
@@ -1,3 +1,33 @@
 Use to display the date. Year will not display if it is the current year.
 
 `DateTime` defaults to the "America/New_York" timezone. `Date` ignores timezone.
+
+**Please note**: this kit could potentially change the dates on you!
+
+This is because the Javascript `Date` class does not have the concept of a date without time. If you pass a Ruby `Date` string to JavaScript, you will receive a date *and* a time which makes use of the timezone it infers from your browser based on your virtual geolocation or custom browser settings.
+
+For example, if you pass a date like "7/2/2022" and your browser is on London time, Javascript will convert this to "7/2/2022 at 0:00 BST".
+
+Subsequently, if you have not passed in a timezone to this kit, it will convert that time into US Eastern Time (default), making the resulting date and time "7/1/2022 at 19:00 EST". In this case the expected July 2 date will now be presented as July 1.
+
+One way around this is to instead pass a Ruby `DateTime` string into this kit which will allow JavaScript to capture the correct timezone.
+
+
+**Ruby**
+```ruby
+my_date = DateTime.new(2021, 07, 02, 18, 0, 0).in_time_zone("London").utc.to_s
+
+# > "2021-07-02 19:00:00 UTC"
+# then provided as prop to React...
+```
+
+**React**
+```jsx
+import { Date as FormattedDate } from "playbook-ui"
+
+<FormattedDate
+    showDayOfWeek
+    value={my_date}
+    {...props}
+/>
+```
```

Comment:
@that-jill I added this part as WIP. I cannot get the output I am expecting, but instead get the browser default which makes the timestamp `13:00` since I am in Central. Can you provide an example of the date-time string to be used from the Ruby output?

Reply:
Not 100% sure what you're asking here.......We solved ours a bit differently. Basically, we wanted to use graphql to validate that what were were passing back and forth dates as opposed to strings (which could potentially allow for weird non-date/time data). So we basically used momentjs to take that date and put it into the timezone we wanted.

ie:

<img width="357" alt="Screen Shot 2021-08-17 at 4 32 27 PM" src="https://user-images.githubusercontent.com/16630021/129796493-4f7408bf-8327-464f-bbb6-312169267dc1.png">

Reply:
ah I see.. `moment` is what's missing in my example